consul_exporter.go

```diff
@@ -397,6 +396,7 @@ func main() {
 		log.Fatalln(err)
 	}
 
+	//lint:ignore SA1019 relax staticcheck verification.
```

Comment:
This can go now. (Just resolve the merge conflict to see why.)

Reply:
Yup, thanks, done.